dist/cookie/{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM crowdbotics/cb-django:3.8-slim-buster AS build
 
 # Copy dependency management files and install app packages to /.venv
 COPY backend/Pipfile backend/Pipfile.lock /
-COPY backend/modules/ backend/modules/
+COPY backend/modules/ /backend/modules/
```

Comment:
What does this do? Basically, what we are doing in this docker stage is  just setting up the environment with the dependency with package installation. We are not referencing the modules here.

Reply:
https://github.com/crowdbotics/modules/pull/806/files/f473567cb214f7f6b8341844bdc41397d9250235..434126d92d5d76df6ff20dee606405c09dd1cb45#diff-72ccc8019942595935d82f368b18dae680e56a08f07096e1ae717ccf75303c8fR6
this line is installing the Pipfile. Under the Pipfile , the Django packages installed on the App are referenced like this:
`cb-django-articles = {editable = true, path = "./modules/django_articles"}  ` . As these packages came from a folder, we need to copy them to be available in the environment.

Reply:
[here](https://github.com/crowdbotics-dev/juan15022023-dev-63826/blob/master/backend/Pipfile) is a Pipfile as example